cmdx.py

```diff
@@ -5247,21 +5417,51 @@ def uninitializePlugin(plugin):
 
 tAddDoubleLinear = om.MTypeId(0x4441444c)
 tAddMatrix = om.MTypeId(0x44414d58)
+tAimConstraint = om.MTypeId(0x44414d43)
 tAngleBetween = om.MTypeId(0x4e414254)
-tBlendShape = om.MTypeId(0x46424c53)
-tMultMatrix = om.MTypeId(0x444d544d)
 tAngleDimension = om.MTypeId(0x4147444e)
 tBezierCurve = om.MTypeId(0x42435256)
+tBlendColors = om.MTypeId(0x52424c32)
+tBlendTwoAttr = om.MTypeId(0x41424c32)
+tBlendShape = om.MTypeId(0x46424c53)
+tBlendWeighted = om.MTypeId(0x41424c57)
+tComposeMatrix = om.MTypeId(0x58000301)
 tCamera = om.MTypeId(0x4443414d)
 tChoice = om.MTypeId(0x43484345)
 tChooser = om.MTypeId(0x43484f4f)
 tCondition = om.MTypeId(0x52434e44)
+tCluster = om.MTypeId(0x46434c53)
+tDecomposeMatrix = om.MTypeId(0x58000300)
+tDeltaMush = om.MTypeId(0x444c544d)
+tDistanceBetween = om.MTypeId(0x44444254)
+tFfd = om.MTypeId(0x46464644)
+tFile = om.MTypeId(0x52544654)
+tFourByFourMatrix = om.MTypeId(0x4642464d)
+tJoint = om.MTypeId(0x4a4f494e)
+tLattice = om.MTypeId(0x464c4154)
+tLocator = om.MTypeId(0x4c4f4354)
 tMesh = om.MTypeId(0x444d5348)
+tMultiplyDivide = om.MTypeId(0x524d4449)
+tMultDoubleLinear = om.MTypeId(0x444d444c)
+tMultMatrix = om.MTypeId(0x444d544d)
+tNetwork = om.MTypeId(0x4e54574b)
+tNonLinear = om.MTypeId(0x464e4c44)
 tNurbsCurve = om.MTypeId(0x4e435256)
 tNurbsSurface = om.MTypeId(0x4e535246)
-tJoint = om.MTypeId(0x4a4f494e)
+tOrientConstraint = om.MTypeId(0x444f5243)
+tPairBlend = om.MTypeId(0x4150424c)
+tPlusMinusAverage = om.MTypeId(0x52504d41)
+tPointConstraint = om.MTypeId(0x44505443)
+tPointOnCurveInfo = om.MTypeId(0x4e504349)
+tPointOnSurfaceInfo = om.MTypeId(0x4e505349)
```

Comment:
Hm, I'm not convinced we need to add this many types to the module, when there is the option to use `cmdt` alongside `cmdx`. What do you think?

Reply:
oh yeah no problem. i missed that one

so why don't you completely remove these ones from cmdx? you could just import them all from cmdt then

Reply:
Because I'd like cmdx to be standalone for most people, but extensible for the power users. Some types, like `tTransform` is more common than `tPairBlend` for example. It's not written in stone.. maybe one day we'll either merge all of them into `cmdx.py` or make a Python package instead.

Reply:
no problem. i'm revising my wrapper anyway so i'll drop all this extra code
i agree on the point everyone should be able to implement this lib the way they want